### ISSUE TITLE
No connection phone via usb cable and Wifi

### ISSUE THREAD

**Issue by bibsn** [rivafarabi/deckboard#395]
Dear developers,
I try to connect my phone via WiFi and also via USB cable with the standard application (not premium) to the DeskApp, but in both cases I received "The device can't connect to your computer. Please try again". 
When I use the USB connection the "scan device" doesn't find any phone. the device was setting in "transfer data" and also in "PTP" for pictures.
Can you help me?
Simone

**Comment by RisedSky** [comment]
Did you check the firewall on your PC ?

![image](https://user-images.githubusercontent.com/23197999/79636575-f8e46e00-8178-11ea-9836-fcc377456515.png)

![image](https://user-images.githubusercontent.com/23197999/79636581-04d03000-8179-11ea-8c31-e718becfbdd7.png)

**Comment by bibsn** [comment]
Thanks for your reply.
but I have already checked my firewall (is one of To DO that I found on the internet).
![image](https://user-images.githubusercontent.com/52285025/79992091-55b29200-84b3-11ea-841a-d30a7714d368.png)
But I want to share with you that last week, a friend of mine, has installed in his old phone (Samsung A3 at 2014) the app and with that the program works, slowly but works.
So I think that the problem isn't in the window's app.

Simone

**Comment by RisedSky** [comment]
Re,
I see here 
![image](https://user-images.githubusercontent.com/23197999/80102501-18add480-8573-11ea-8f8f-21694f03c499.png)
that you didn't put a check mark, can you put it, restart your computer and check if it works correctly please ?
Also, be sure that you're in the correct wifi, and not in a guest one (you can split a wifi in 2 with the master one and the other one that is a "guest", the guest one is not in the same network as the master one)
Regards

**Comment by bibsn** [comment]
Dera RisedSky
I set, as you suggested, to allow the app to communicate trough Windows Defender Firewall, but nothing changed. 
![image](https://user-images.githubusercontent.com/52285025/80274559-a4db0b80-86db-11ea-9f23-84928a0dd4cd.png)
I have the problem trough the Wi-Fi but also via the USB cable. in the second case, the Deckboard APP doesn't found any device.
I wrote in the previous post that "But I want to share with you that last week, a friend of mine, has installed in his old phone (Samsung A3 at 2014) the app and with that the program works, slowly but works." with the previous setting.
Regards.
Simone

**Comment by RisedSky** [comment]
Can you try to disable your firewall when you try to connect ? (just to give it a try)

**Comment by christerjohansson** [comment]
I have this very same issue. Can't connect at all. I've tried disabling the firewall completely, running the app as adin. But computer then rejects connect...meaning there is no service running on default port 80. 

I believe you need to implement an alternate port to run the webserver at, and not the default 80.

**Comment by rivafarabi** [comment]
For anyone experiencing this issue. Please update to version 1.9.52.

In the previous versions, the IP address list in _Possible IP Addresses_  modal has been filtered. The list only shows the addresses with _192.168.xxx.xxx_ pattern.

This filter has been disabled in the latest version. So user can try connecting the app through all IP addresses that was hidden before.

Let me know if the issue still occurs

**Comment by christerjohansson** [comment]
> For anyone experiencing this issue. Please update to version 1.9.52.
> 
> In the previous versions, the IP address list in _Possible IP Addresses_ modal has been filtered. The list only shows the addresses with _192.168.xxx.xxx_ pattern.
> 
> This filter has been disabled in the latest version. So user can try connecting the app through all IP addresses that was hidden before.
> 
> Let me know if the issue still occurs

The issue still persists. You will need to make it possible to change ports for the application. Port 80 is not recommended, please set alternate port for this application or make it an option in settings.

**Comment by rivafarabi** [comment]
The server app is running on port 8500, not 80. Make sure the firewall allow the connection on port 8500

**Comment by christerjohansson** [comment]
> The server app is running on port 8500, not 80. Make sure the firewall allow the connection on port 8500

Ok, I do get connection through 8500. 
But now it says 404, and shows "Cannot GET /"

Console outputs: "Refused to load the image 'http://127.0.0.1:8500/favicon.ico' because it violates the following Content Security Policy directive: "default-src 'none'". Note that 'img-src' was not explicitly set, so 'default-src' is used as a fallback."

Same result on IP 192.168.x.x.

**Comment by ReyTrollG4m3r** [comment]
Hello, I am curious to know if you have a tutorial on how to connect a cell phone deckboard to a PC deckboard via usb. It is that I did not understand very well the explanations that the app gives you when you go to the help section.

**Comment by ttv-notnvs** [comment]
> Hello, I am curious to know if you have a tutorial on how to connect a cell phone deckboard to a PC deckboard via usb. It is that I did not understand very well the explanations that the app gives you when you go to the help section.

Hi!

I may have found a solution for this but it involves another app for it to work:

TLDR: somehow open USB ports with some software and relaunch Deckboard. 

Long explanation:
I stream on twitch and while trying to find a good webcam, I've realized that using an old smartphone for it would be a free and a simple solution so I downloaded DroidCam (this is not a self promoting thing nor a commercial thing, just helps explains the solution I've found)

So, I've installed the app on the phone as well as the app on Windows (11 for reference).
For it to work, I'm prompted with an initial configuration for (I think) open USB ports (screenshot attached).
On the list, clicked refresh and I got the smartphone and the tablet. I select the smartphone since I'm using it as a webcam.

After this, simply open Deckboard on PC and on the tablet, select connect via USB and you may be good to go!

Hope it helps!

![image](https://user-images.githubusercontent.com/120224602/206804848-6455fca3-8e55-4d4b-82cb-4cea6f59dd05.png)

**Comment by TioGaspar** [comment]
I find the solution in this [video from Youtube](https://www.youtube.com/watch?v=5TqRbN4UTrM), if you don't trust the link search like this **"Criando um StreamDeck via DeckBoard e tablet - passo a passo"**
